### PR TITLE
Add types

### PR DESCRIPTION
**Pull request by LcpMarvel**
Please release a new version.

https://github.com/wakirin/Litepicker/issues/9

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
     "outDir": "./dist/js",
     "module": "commonjs",
     "target": "es6",
+    "declaration": true,
+    "downlevelIteration": true,
```
Comment (wakirin) [inline tsconfig.json]: I am not sure we really need `downlevelIteration`. Project is target to es6, we already use class, spread operator and etc.

File: src/litepicker.ts [hunk 2]
```diff
@@ -333,7 +344,8 @@ export class Litepicker extends Calendar {
 
       if (this.options.splitView) {
         const monthItem = target.closest(`.${style.monthItem}`);
-        idx = [...monthItem.parentNode.childNodes].findIndex(el => el === monthItem);
```
Comment (wakirin) [inline src/litepicker.ts]: What is wrong with `findIndex` ? Why you are create a custom function ?

File: tsconfig.json (hunk 1 above)
Comment (LcpMarvel) [reply tsconfig.json]: yes, I previously set target to `es5`. so I add this option and update spread operator.

File: src/litepicker.ts (hunk 2 above)
Comment (LcpMarvel) [reply src/litepicker.ts]: spread operator used here is not good.

**Comment by LcpMarvel** [comment]
updated, it's your choice, I just want the types

File: src/litepicker.ts (hunk 2 above)
Comment (wakirin) [reply src/litepicker.ts]: Could you explain why ?

**Pull request merged**